apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/internal/DefaultCompilationUnit.kt

```diff
@@ -204,28 +209,32 @@ class DefaultCompilationUnit(
       )
     }
 
-    fun fromFiles(project: Project, apolloExtension: DefaultApolloExtension, apolloVariant: ApolloVariant): List<DefaultCompilationUnit> {
+    fun fromFiles(project: Project, apolloExtension: DefaultApolloExtension, apolloVariant: ApolloVariant): DefaultCompilationUnit {
       val sourceSetNames = apolloVariant.sourceSetNames
       val schemaFiles = findFilesInSourceSets(project, sourceSetNames, ".") {
         it.name == "schema.json"
       }
+      require(schemaFiles.size == 1) {
+        """|
+          |By default only one schema.json file is supported. If you have multiple schema.json,
+          |Please use a service instead:
```

Comment:
Maybe "Please use multiple services instead" ?

Reply:
Makes a lot of sense 💯

Reply:
With that, I will also change the example just below to have 2 services.

Reply:
From the test, here is how it will look like: 

```
A problem occurred configuring root project 'testProject'.
> By default only one schema.json file is supported.
  Please use multiple services instead:
  apollo {
    service("serviceName") {
      sourceFolder = "com/example/abc/"
    }
    service("serviceName") {
      sourceFolder = "com/example/"
    }
  }
```